bigquery/bigquery_reservation_quickstart/main.go

```diff
@@ -26,48 +26,57 @@ import (
 	"fmt"
 	"log"
 
-	bqReservation "cloud.google.com/go/bigquery/reservation/apiv1"
+	reservationapi "cloud.google.com/go/bigquery/reservation/apiv1"
 	"google.golang.org/api/iterator"
 	reservationpb "google.golang.org/genproto/googleapis/cloud/bigquery/reservation/v1"
 )
 
 // Define two command line flags for controlling the behavior of this quickstart.
 var (
-	projectID = flag.String("project_id", "",
-		"Cloud Project ID, used for session creation.")
-	location = flag.String("location", "US",
-		"BigQuery location used for interactions")
+	projectID = flag.String("project_id", "", "Cloud Project ID, used for session creation.")
+	location  = flag.String("location", "US", "BigQuery location used for interactions")
 )
 
 func main() {
 	flag.Parse()
+	validateFlags()
 	ctx := context.Background()
-	bqResClient, err := bqReservation.NewClient(ctx)
+	bqResClient, err := reservationapi.NewClient(ctx)
 	if err != nil {
 		log.Fatalf("NewClient: %v", err)
 	}
 	defer bqResClient.Close()
 
-	b, err := printCapacityCommitments(ctx, bqResClient)
+	s, err := reportCapacityCommitments(ctx, bqResClient, *projectID, *location)
 	if err != nil {
 		log.Fatalf("printCapacityCommitments: %v", err)
 	}
-	fmt.Println(string(b))
+	fmt.Println(s)
 
-	b, err = printReservations(ctx, bqResClient)
+	s, err = reportReservations(ctx, bqResClient, *projectID, *location)
 	if err != nil {
 		log.Fatalf("printReservations: %v", err)
 	}
-	fmt.Println(string(b))
+	fmt.Println(s)
+}
+
+// validateFlags does some basic validation of the command line flags.
+func validateFlags() {
```

Comment:
I think this could be inlined? Up to you. Also, generally not a fan from calling `log.Fatal` in a function, which is another benefit of inlining (can still call `log.Fatal` directly).

Reply:
inlined